docs-generation/scripts/batch_translate_docs.py

```
@@ -139,50 +138,52 @@ def strip_ai_preamble(content: str, lang: str) -> str:
 
     Works by:
     1. Extracting output names from en.md Outputs table in order (row by row).
-    2. Extracting output table lines from translated content.
+    2. Extracting output data rows from translated content (rows whose first cell
+       is a backtick-wrapped name).
     3. Replacing the first column (output name) of each row with its English counterpart.
     """
     # Extract English output names
     en_outputs_match = re.search(
-        r'##\s+(?:Outputs|输出|輸出|出力|출력|Выходы|Salidas|Sorties|المخرجات|Çıktılar|خروجی‌ها)\s*\n\n(.*?)(?=\n##|\Z)',
+        r'##\s+(?:Outputs|输出|輸出|出力|출력|Выходы|Salidas|Sorties|المخرجات|Çıktılar|خروجی‌ها)\s*\n(.*?)(?=\n##|\Z)',
```

Ruff (0.16.1)

[warning] String contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)